pkg/apis/github.com/presslabs/mysql-operator/pkg/apis/mysql/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,450 @@
+// +build !ignore_autogenerated
```

Comment:
Remove this file. This file is generated incorrectly because `GOPATH` was not set. Before running `make`  make sure that `GOPATH` is set.